controllers/provisioners/eks/eks_test.go

```diff
@@ -129,6 +129,7 @@ func MockInstanceGroup() *v1alpha1.InstanceGroup {
 		Spec: v1alpha1.InstanceGroupSpec{
 			Provisioner: ProvisionerName,
 			EKSSpec: &v1alpha1.EKSSpec{
+				Type:    "LaunchConfiguration",
```

Comment:
do we ever test the default case of `Type` being undefined?

Reply:
This happens in instancegroup_types.go:360, we SHOULD add unit tests around all of the validations we do, but since this is a larger scope and this is manually tested, I will create an issue to address this

Reply:
#192 opened